templates/settings.py.erb

```diff
@@ -19,9 +19,14 @@ DATABASES = {
 <% end -%>
     },
 }
+
+<% if scope['pulpcore::configure_redis_connection'] -%>
 REDIS_HOST = "localhost"
 REDIS_PORT = "<%= scope['redis::port'] %>"
 REDIS_DB = <%= scope['pulpcore::redis_db'] %>
+<% end -%>
+
```

Comment:
Slight nit: this leaves 2 empty lines in the generated file so I'd suggest to include empty line inside the if/end block. Github doesn't render this well, but roughly
```suggestion

<% end -%>
```